### ISSUE TITLE
Air Gap Installation throws Node JS error on Windows 11 with no internet connectivity

### ISSUE THREAD

**Issue by cssmith4** [containers/podman-desktop#7779]
### Bug description

![image](https://github.com/containers/podman-desktop/assets/26615653/b81c2e1f-afe3-466a-8b94-5576d9254107)

Node 18.20.2 is installed on local machine.  Leverage this installer podman-desktop-airgap-1.10.3-setup-x64.exe.  The error message will not go away.  Hit OK, over 30 times and the message reappears.  Podman Desktop is visible behind the error message.  I killed podman desktop and the error message will not go away.  

### Operating system

Windows 11 Enterprise 22631.3737

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Steps to reproduce

Run podman-desktop-airgap-1.10.3-setup-x64.exe installer on windows 11 machine (see additional context for versioning).  I have feeling issue is related to either my Node version on machine or no internet connectivity.

### Additional context

12th Gen Intel(R) Core(TM) i5-1245U   1.60 GHz
WSL 2.1.5 installed
Node 18.20.2 installed
Edition	Windows 11 Enterprise
Version	23H2
OS build	22631.3737
Experience	Windows Feature Experience Pack 1000.22700.1009.0

**Comment by lstocchi** [comment]
Tried to replicate on my machine by downgrading node without any success. Windows and WSL have the same version/build.

**Comment by odockal** [comment]
Tried on Windows 10 with no internet connection. although I had node 20 installed. I did not hit the issue.

**Comment by ScrewTSW** [comment]
@cssmith4 Hello. I tried to reproduce this issue.

Unfortunately I do not have Windows 11 on my local machine, and as far as I can see that's the only difference in my reproduction strategy.

I tried your exact versions of the airgap installer and NodeJS.msi installer from nodejs.org

I also tried the latest v18 LTS installer and the latest airgap installer (v1.11.1)
I also tried it with latest node version installed using `nvm-windows` and the v18 LTS and v18.20.2 from there.

I did not see any error message pop up.
